pkgs/applications/science/biology/sortmerna/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, cmake, rocksdb, rapidjson, pkgconfig, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sortmerna";
+  version = "3.0.3";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "biocore";
+    rev = "v${version}";
+    sha256 = "0zx5fbzyr8wdr0zwphp8hhcn1xz43s5lg2ag4py5sv0pv5l1jh76";
+  };
+
+  patches = [ ./CMakeInstallPrefix.patch ];
```

Comment:
This was commented out in a commit upstream https://github.com/biocore/sortmerna/commit/4d36d620a3207e26cf3f588d4ec39889ea21eb79.

Can you `fetchpatch` it?
> https://nixos.org/nixpkgs/manual/#sec-patches